### DISCUSSION TITLE
Share documentation for performance related metrics higress supports

### DISCUSSION THREAD

**Discussion by pravinpushkar** [alibaba/higress#307] [Q&A]
I would like to understand the various performance related metrics that higress promises as if now like QPS, Average Latency etc.
May be some sort of benchmarking data that would also be useful for users ?

Please help me with any documentation if there is already available. Thanks

**Comment by pravinpushkar** [comment]
Hello, bumping up the thread again, Sorry. Please let me know if there is any clarifying question?

**Reply by johnlanni** [reply]
Hi @pravinpushkar, sorry for the late reply. Many times we pay more attention to functions than performance, because Higress is based on the Envoy engine, and performance is not a problem in most scenarios. For simple scenarios, Benchmark can directly refer to the performance of Envoy. I don’t know which scenarios you are more interested in Benchmark, such as wasm plugins?

**Reply by pravinpushkar** [reply]
@johnlanni Thanks for the response.
> Higress is based on the Envoy engine, and performance is not a problem in most scenarios. For simple scenarios, Benchmark can directly refer to the performance of Envoy

Understood. Thanks
 
> I don’t know which scenarios you are more interested in Benchmark, such as wasm plugins?
 
 I was mainly looking at things line QPS, Avg latency etc. numbers. But I think these numbers should be comparable to Envoy from above statements.
